sdk/core/core-http/lib/policies/exponentialRetryPolicy.ts

```diff
@@ -50,6 +50,42 @@ const DEFAULT_CLIENT_RETRY_COUNT = 3;
 const DEFAULT_CLIENT_MAX_RETRY_INTERVAL = 1000 * 90;
 const DEFAULT_CLIENT_MIN_RETRY_INTERVAL = 1000 * 3;
 
+/**
+ * Options that control how to retry failed requests.
+ */
+export interface RetryOptions {
+  /**
+   * The maximum number of retry attempts.  Defaults to 3.
+   */
+  retryCount?: number;
```

Comment:
This should be `maxRetries` as per the retry guidelines that @bterlson had shared before

Reply:
Thanks!  Will fix it